templates/generic_show_form.html.twig

```diff
@@ -78,6 +78,16 @@
                         ) }}
                      {% endif %}
 
+                     {% if item.isField('template_name') and withtemplate == 1 %}
```

Comment:
This field is already present in `templates/components/form/header_content.html.twig`. Maybe this field should only be present `{% if not include_header_content %}`, but I am not really sure how it should be handled.